website/source/docs/internals/architecture.html.md

```diff
@@ -22,10 +22,10 @@ to learn about them without having to go spelunking through the source code.
 Before describing the architecture, we provide a glossary of terms to help
 clarify what is being discussed:
 
-* Agent - An agent is the long running daemon on every member of the Consul cluster.
+* Agent - An agent is the long running daemon on every node of the Consul cluster.
 It is started by running `consul agent`. The agent is able to run in either *client*
 or *server* mode. Since all nodes must be running an agent, it is simpler to refer to
-the node as being either a client or server, but there are other instances of the agent. All
+them as being either a client or server, but there are other instances of the agent. All
```

Comment:
`but there are other instances of the agent`

I don't understand this at all. I know it's not your addition but I suggest:

```suggestion
them as being either a client or server. All
```